src/lib/db.js

```diff
@@ -35,6 +36,8 @@ function executeSQLPlus (sqlFilepath) {
     const command = '/opt/oracle/instantclient/sqlplus'
     const args = [url, '@' + sqlFilepath]
     const process = spawn(command, args, {env})
+    process.stdout.on('data', (data) => console.log(data.toString()))
```

Comment:
I don't know if we always want to show all this; could we send it to the debug logger so it is only shown in debug mode?